test/v1alpha1/cluster_resource_test.go

```diff
@@ -140,13 +140,13 @@ clusters:
 - cluster:
     certificate-authority-data: WTJFdFkyVnlkQW89
     server: https://1.1.1.1
-  name: helloworld-cluster
+  name: target-cluster
 contexts:
 - context:
-    cluster: helloworld-cluster
+    cluster: target-cluster
     user: test-user
-  name: helloworld-cluster
-current-context: helloworld-cluster
+  name: target-cluster
+current-context: target-cluster
```

Comment:
i dont totally understand why this changed - i thought we were changing the fields used in status, this seems to imply we might be making a backwards incompatible change to the behavior of the cluster resource? lemme know if im wrong tho

Reply:
Yeah I guess this is a separate bug being fixed as a side effect - unsure of the best way to handle it. The resource in this case was named "target-cluster" in the task, the directory it was written to always should have been target-cluster. 

It looks like all the other resources determine where to write data using the "path" provided to GetInputTaskModifier, but the cluster resource used the incorrect name field.

People that referenced cluster resources using interpolation like `$(resources.inputs.target-cluster)` should see no change, but those that hardcoded it like this test did would see a breakage.

Any suggestions?